### DISCUSSION TITLE
Extension to AttachCommand

### DISCUSSION THREAD

**Discussion by SFBots313** [Zrips/CMI#2555] [Q&A]
First want to start off by saying this is an amazing plugin I find more and more utility from everyday, love to be able to uninstall multiple other plugins when I find CMI does it better and usually easier to do it with.

My Suggestion ( Or question of how to do it, if its already possible ):

Im trying to make a "Repair Kit" item that when dropped onto another item within the players inventory performs a command on it, in this case /repair.  This could be further useful for enchanting something through the same means like many enchantment plugins do where you can drag a custom enchant book onto something and itll enchant it.

So far I have:
/cmi attachcommand !limiteduse:1! ;;!cc! /cmi repair hand

To my understand this command attached to the custom item I created will perform the /repair hand command as console, with a single usage and the item will disappear. Which all works fine, but you cant right click this item to perform the command and have the item in your hand at the same time. 

Like I said before im not 100% sure this isnt already possible but ive read through the tutorial and googled about it quite bit trying to find my answer, and asked in the Zrips Discord with no luck.

Appreciate any help and Love your fine work Zrips!

Thanks

**Comment by xsmeths** [comment]
idk about exactly what you wanted but a workaround could be to repair all and make it disappear, just thought I'd suggest it as i noticed the report

**Comment by SFBots313** [comment]
Appreciate the help, I am aware of this, just not really what I want this to do.  Prob just gonna use a repair sign, if I can't get this to work.  Though I'm sure other plugins offer something that can do this, I just like using CMI to its fullest potential.
